src/Selenium2Library/keywords/element.py

```diff
@@ -141,26 +152,30 @@ def locator_should_match_x_times(self, locator, expected_locator_count, message=
         See `Page Should Contain Element` for explanation about `message` and
         `loglevel` arguments.
         """
-        actual_locator_count = len(self._element_find(locator, False, False))
+        actual_locator_count = len(self.element_find(
+            locator, first_only=False, required=False)
+        )
```

Comment:
This is really hard to read. I'd use

```python
actual_locator_count = len(self.element_find(locator, first_only=False,
                                             required=False))
```

or

```python
elements = self.element_find(locator, first_only=False, required=False)
actual_locator_count = len(elements)
```

instead. Shortening the local `actual_locator_count` variable to just `count` would probably be a good idea too. Everything might be nicely < 80 chars after that.